util-core/src/main/scala/com/twitter/util/Try.scala

```diff
@@ -67,6 +80,11 @@ object Try {
  */
 sealed abstract class Try[+R] {
   /**
+   * Convert to a scala.util.Try
+   */
+  def asScalaTry: scala.util.Try[R]
```

Comment:
(bikeshed, so feel free to drop)

what do you think of shortening this to `asScala` since this going to be something called frequently? (ditto for `fromScala`)